### ISSUE TITLE
Misleading instructions for uploading public key

### ISSUE THREAD

**Issue by b-jazz** [keybase/keybase-issues#1936]
When I go to upload my **public** key, the page suggests the following commands to display my public key:

    # tip: first list your keys in GPG
    gpg -K --keyid-format long --with-colons --with-fingerprint
    # then export the one you want (look next to `fpr`)
    gpg --export -a A4AA3A5BDBD40EA549CABAF9FBC07D6A97016CB3

The first command uses the long form arguments for everything except the `-K`, which checking the man page tells me it is for listing the **secret** key, not the **public** key. As far as I can tell, there is only one fingerprint for the public/secret pair, so there is no need to use `-K` or `--list-secret-keys` and the instructions could be changed to the following:

    # tip: first list your keys in GPG
    gpg --list-keys --keyid-format long --with-colons --with-fingerprint

**Comment by zQueal** [comment]
Yes, this seems to be a typo. To output your public key you'd use `gpg -k` and not `gpg -K`. However, you'd be displaying all other imported keys, too, not just your own.

    λ gpg -k --keyid-format long --with-fingerprint | grep 8516
    pub   4096R/B1DA56DCB73E8516 2014-03-28 [expires: 2017-12-31]
          Key fingerprint = 7B5A C030 0E2C 74FD 355C  CC81 B1DA 56DC B73E 8516
